coral/src/app/features/components/filters/MyRequestsFilter.tsx

```diff
@@ -0,0 +1,24 @@
+import { Switch } from "@aivenio/aquarium";
+import { useFiltersValues } from "src/app/features/components/filters/useFiltersValues";
+
+function MyRequestsFilter() {
+  const { showOnlyMyRequests, setFilterValue } = useFiltersValues();
+
+  const handleChangeIsMyRequest = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    if (event.target.checked) {
```

Comment:
Couldn't that be `setFilterValue( { name: "showOnlyMyRequests", value: event.target.checked })`?

Reply:
`event.target.checked` is a `boolean`, so it would fit, but `String(event.target.checked)` would definitely be better than the current form :o

Reply:
Actually I think stringifying boolean values is super prone to confusing moments, so I changed the way we handle it by stringifying exactly in the spot where we need to (when setting the url search param), and keeping `boolean` everywhere else :o [848006b](https://github.com/aiven/klaw/pull/903/commits/848006ba63e94f16c35699c930ccdc893c337e87)

Reply:
![jyes](https://user-images.githubusercontent.com/20607294/227871685-3e3a50be-8055-4c36-94ad-5d8a27173eb4.gif)